web/Services/AdminService.cs

```diff
@@ -48,21 +48,23 @@ private async Task<List<UserDbModel>> GetCurrentAdmins()
 
     private async Task<List<UserDbModel>> GetPendingAdmins()
     {
-        var pendingAdmins =
-            await _dbContext.Users.Where(u => u.IsAdmin == false && u.HasRequestedAdmin).ToListAsync();
-        return pendingAdmins;
+        List<UserDbModel> PendingAdmins = await _dbContext.Users.Where(u => u.IsAdmin == false && u.HasRequestedAdmin).ToListAsync();
```

Comment:
Do you not get a warning on this line about `PendingAdmins` not meeting the naming convention?

Reply:
Yes, I'm not quite sure how that slipped through before.